src/ProcessManager.php

```diff
@@ -619,11 +619,11 @@ protected function commandReady(array $data, ConnectionInterface $conn)
 
         if ($this->allSlavesReady()) {
             if ($this->status === self::STATE_EMERGENCY) {
-                $this->output->writeln("<info>Emergency survived. Workers up and running again.</info>");
+                $this->output->writeln('<info>Emergency survived. Workers up and running again.</info>');
```

Comment:
Imho this is a valid change but will make it more complicated if we decide later to add variables.

Reply:
Why not add variable using sprintf?